docs/installation.md

```diff
@@ -83,21 +83,25 @@ It is not possible to reset this setting for all existing projects.
 
 ```scala
 // In project/plugins.sbt. Note, does not support sbt 0.13, only sbt 1.0.
-addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.0-RC5")
```

Comment:
This should be `@STABLE_VERSION@` so that it updated automatically with new releases, it was hardcoded to 1.5.1 because the sbt plugin hadn't been released yet for v2.